assets/tasks/common.json

```diff
@@ -943,7 +949,9 @@
                 "-t",
                 "cross-toolchain-riscv64-__container__",
                 "--build-arg",
-                "IMAGE_ARCH=riscv64"
+                "IMAGE_ARCH=riscv64",
+                "--build-arg",
+                "GPU=${config:torizon_gpu}"
```

Comment:
@andreriesco please drop this commit, these are not needed for the SDK containers, the linker will link the binary with the generic lib, and the specific GPU ones will be patched on the runtime container

Reply:
I think it makes sense to keep just the arm64 one, as it uses the "-vivante" containers, which may have differences from the upstream even when compiling (like the libGLX support for example)